cross-chain/polygon/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@keep-network/tbtc-v2-polygon",
+  "version": "1.0.0-dev",
+  "description": "tBTC v2 on Polygon",
+  "license": "GPL-3.0-only",
+  "files": [
+    "artifacts/",
+    "build/contracts/",
+    "contracts/",
+    "!contracts/hardhat-dependency-compiler",
+    "!**/test/",
+    "parentchain/",
+    "sidechain/",
```

Comment:
Shouldn't we have `deploy_parentchain` and `deploy_sidechain` here instead?

Reply:
We should [03837dc](https://github.com/keep-network/tbtc-v2/pull/588/commits/03837dc2418c4027edc192727a6d2f025bc4d5e9)